pkg/storaged/stratis-details.jsx

```diff
@@ -398,6 +547,56 @@ export const StratisPoolDetails = ({ client, pool }) => {
                         </DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    { pool.Encrypted && client.features.stratis_crypto_binding &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm className="control-DescriptionListTerm">
+                            {_("storage", "Passphrase")}
+                        </DescriptionListTerm>
+                        <DescriptionListDescription>
+                            { key_desc ? cockpit.format(_("using key description \"$0\""), key_desc) : <i>{_("not configured")}</i> }
+                            <DescriptionListDescription className="tab-row-actions">
+                                { !key_desc
+                                    ? <StorageButton onClick={add_passphrase}>{_("Add")}</StorageButton>
+                                    : null
+                                }
+                                { key_desc
```

Comment:
These two cases could be in one brace, as an either-or.

Reply:
Indeed! Done.